packages/perspective/src/js/perspective.js

```diff
@@ -305,6 +305,7 @@ export default function(Module) {
         }
 
         const get_pkeys = !!options.index;
+        const get_ids = !!options.id;
```

Comment:
`id` is IMO a really good solution to this problem we had previously special cased with nasty logic.  Some docs on what `id` does, both in the module API and the turtorial/manual/website, would be excellent.